redisinsight/templates/statefulset.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
**suggestion:** I suggest to use a `Deployment` here, given we have control to change this.